INFO  01:23:19 Application started. Press Ctrl+C to shut down.
INFO  01:23:19 Hosting environment: Development
INFO  01:23:19 Content root path: C:\Users\user\source\repos\AuthenticationApi\AuthenticationApi
INFO  03:46:13 Application started. Press Ctrl+C to shut down.
INFO  03:46:13 Hosting environment: Development
INFO  03:46:13 Content root path: C:\Users\user\source\repos\AuthenticationApi\AuthenticationApi
INFO  05:12:48 Application started. Press Ctrl+C to shut down.
INFO  05:12:48 Hosting environment: Development
INFO  05:12:48 Content root path: C:\Users\user\source\repos\AuthenticationApi\AuthenticationApi
INFO  09:04:40 Application started. Press Ctrl+C to shut down.
INFO  09:04:40 Hosting environment: Development
INFO  09:04:40 Content root path: C:\Users\user\source\repos\AuthenticationApi\AuthenticationApi
INFO  09:10:12 Application started. Press Ctrl+C to shut down.
INFO  09:10:12 Hosting environment: Development
INFO  09:10:12 Content root path: C:\Users\user\source\repos\AuthenticationApi\AuthenticationApi
INFO  09:14:37 Application started. Press Ctrl+C to shut down.
INFO  09:14:37 Hosting environment: Development
INFO  09:14:37 Content root path: C:\Users\user\source\repos\AuthenticationApi\AuthenticationApi
INFO  12:13:47 Application started. Press Ctrl+C to shut down.
INFO  12:13:47 Hosting environment: Development
INFO  12:13:47 Content root path: C:\Users\user\source\repos\AuthenticationApi\AuthenticationApi
INFO  12:47:58 Application started. Press Ctrl+C to shut down.
INFO  12:47:58 Hosting environment: Development
INFO  12:47:58 Content root path: C:\Users\user\source\repos\AuthenticationApi\AuthenticationApi
ERROR 03:36:22 An unhandled exception has occurred while executing the request.
System.NullReferenceException: Object reference not set to an instance of an object.
   at AuthenticationApi.Controllers.TokenController.AuthenticateUser(Users login) in C:\Users\user\source\repos\AuthenticationApi\AuthenticationApi\Controllers\TokenController.cs:line 73
   at AuthenticationApi.Controllers.TokenController.Login(Users login) in C:\Users\user\source\repos\AuthenticationApi\AuthenticationApi\Controllers\TokenController.cs:line 34
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
ERROR 03:36:22 An unhandled exception has occurred while executing the request.
System.NullReferenceException: Object reference not set to an instance of an object.
   at AuthenticationApi.Controllers.TokenController.AuthenticateUser(Users login) in C:\Users\user\source\repos\AuthenticationApi\AuthenticationApi\Controllers\TokenController.cs:line 73
   at AuthenticationApi.Controllers.TokenController.Login(Users login) in C:\Users\user\source\repos\AuthenticationApi\AuthenticationApi\Controllers\TokenController.cs:line 34
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
INFO  07:35:04 Application started. Press Ctrl+C to shut down.
INFO  07:35:04 Hosting environment: Development
INFO  07:35:04 Content root path: C:\Users\user\source\repos\AuthenticationApi\AuthenticationApi
